octodns_transip/__init__.py

```diff
@@ -135,11 +248,44 @@ def _apply(self, plan):
                 'Unhandled error: ({}) {}'.format(e.response_code, e.message)
             )
 
+        for change in changes:
+            record = change.data['new']
+
+            if change.data['name'] == '' and change.data['record_type'] == 'NS':
+                values = (
+                    record.get('values')
+                    if record.get('values')
+                    else [record.get('value')]
+                )
```

Comment:
Same here, since it's an NS should be able to just do `.values`.

```suggestion
                values = record.values
```

Reply:
Applied and reverted, this broke the tests.
Tried `values = record.get('values')` but that broke when i altered the unit.tests.yaml to have a single value NS record
```
? ''
:  - type: NS
    value: ns0.transip.net.
```

```
                values = record.get('values')

                nameservers = []
>               for value in values:
E               TypeError: 'NoneType' object is not iterable

octodns_transip/__init__.py:268: TypeError
```

Reply:
Ah. I didn't notice that a couple of lines above, ~257 that it's serializing the change using the `.data` property.

```
            record = change.data['new']
```

That's what's causing the problem/complications. That turns everything into a `dict` which loses the all of the `Record` object's behavior.  

If you apply the above change. This one should work correctly. (I've tested it locally now.)